.github/workflows/payment-test-suite.yml

```
@@ -0,0 +1,523 @@
+      - name: Checkout code
+        uses: actions/checkout@v4
+        
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
+          cache: 'npm'
+          
+      - name: Cache dependencies
+        id: cache
+        uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)